release-notes.txt

```diff
@@ -1,3 +1,543 @@
++ panic: callback app during panic
++ bitarithm: add bitarithm_test_and_clear()
+* thread: Add THREAD_NAMES make flag
+* kernel_defines.h: Fix container_of()
+* core: Add and use helper functions for internal data
+* treewide: Fix direct access to scheduler internals
+* core/bitarithm: use __builtin_clz() for bitarithm_msb()
+
+System Libraries
+----------------
++ architecture: New module to access architecture internals
++ Add support for PicoLIBC
++ ztimer: add ztimer_spin()
++ ztimer: add periodic timers API
+* byteorder:
+    * Moved to sys
+    + Add byteorder_bebuftohll() and inverse
+    + add uint32 from/to buffer funcs
++ event: implement waiting on multiple queues
++ evtimer: add mbox support
++ random: add SHA256 mode to SHA1PRNG & tests
+* sys/luid: improve luid_base() algorithm
+* tsrb: Make thread-safe
+* pipe_dynamic: fix possible null pointer dereference
+
+Networking
+----------
++ ieee802154_submac: add initial support for common MAC sub layer
++ ieee802154/radio_hal: introduce Radio HAL for IEEE802.15.4 compatible radios
++ EUI:
+    + luid: add luid_netdev_get_eui48() & luid_netdev_get_eui64()
+    + eui64: add eui64_to_eui48()
++ link_layer: implement EUI provider
+* luid: shuffle bytes
+* uhcp: use LOG_xxx functions
+* gcoap: Insert registrations at the front
+* gnrc_ipv6_nib: only multicast probe UNREACHABLE neighbor if 6LR
+* gnrc_ipv6_nib: set NCE stale on incoming ARO
+* gnrc_ipv6_nib: don't add non-link-local to NC without ARSM
++ gnrc_netif_pktq: add packets to optional send queue when device is busy
+* emcute: fix buffer overflow in _willupd_msg()
+* SUIT:
+    + Introduction of a payload storage API for SUIT manifest payloads
+    * Upgrade to draft-ietf-suit-manifest-09
+    * fail fetch if the image size doesn't match expected
+* gnrc:
+    - Remove cyclic dependencies
+    - Remove deprecated module gnrc_nettest
+- netopt: remove deprecated alias NETOPT_LINK_CONNECTED
+
+Packages
+--------
++ openwsn: add riot based openwsn scheduler
++ openwsn: add shell commands
+* openthread: migrate to event loop
+* nimble/netif: fix deadlock on connection loss
+* wolfssl: bump version to 4.5.0
+* nanopb: bump version to 0.4.3
+* ccn-lite: bump version
+* cn-cbor bump to 1.0.0
+* tinycbor: bump to v0.5.3
+* tinydtls: enforce the selection of a crypto secure PRNG
+* jerryscript: bump to version 2.3.0
+* semtech-loramac: replace xtimer by ztimer
+* lvgl:
+    + add logic for managing touch devices
+    + introduce screen_dev wrapper
+    * bump to v7
+- emb6: remove deprecated package
+
+Boards
+------
++ nrf52: Enable PWM on the LED pins
++ nucleo-g071rb: add initial support
++ nucleo-g070rb: add board
++ nucleo-g431rb: add initial support
++ Support for YARM board
++ add Adafruit Itsy Bitsy
+* samr21-xpro: fix reading EUI-64 from EDBG on cold boot
+* nucleo-l452re: add i2c config
+* hifive1b: add support for openocd programmer
+- stm32f4: remove default clock configuration for 96MHz and 168MHz CPUs
```

Comment:
I don't think this is part of the release

Reply:
You're right, removed